packages/matplotlib/meta.yaml

```diff
@@ -48,6 +46,7 @@ requirements:
   host:
     - numpy
   run:
+    - contourpy
```

Comment:
```suggestion
```

I think I added this in error when testing something 😅

Reply:
Did you mean to remove this? Or should it actually be here?

Reply:
Okay I see @ianthomas23's comment now.